### DISCUSSION TITLE
Darstellungsfilter mit Nachbarschaftsberechnung

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#73] [Q&A]
Ich dachte der Darstellungsfilter wird auch bei der Nachbarschaftsberechnung berücksichtigt. 
Aber so ist es nicht - bei Darstellungsfilter "visible".
Darstellungsfilter "locked" - bei uns werden mit dieser Einstellung Header-parameter ausgelesen und als Filterwerte verwendet - funktioniert.
Bei "locked" kann man nicht über den Umweg "Nachbarschaftsabfrage" die Filterung "aufheben", wie bei "visible".

lg marko

**Comment by gis-eni** [comment]
Die clientseitigen Filter (=Filter, die vom Anwender geändert werden können) wurden bei der Nachbarschaftsberechnung nicht an die API (Server) zurück gegeben. 
Dieses Problem ist mit Build 5.23.2101 behoben
